packages/vector_graphics/test/vector_graphics_test.dart

```diff
@@ -254,22 +254,6 @@ void main() {
     expect(tester.layers, contains(isA<PictureLayer>()));
   });
 
-  testWidgets('Uses repaint boundary', (WidgetTester tester) async {
-    final TestAssetBundle testBundle = TestAssetBundle();
-
-    await tester.pumpWidget(
-      DefaultAssetBundle(
-        bundle: testBundle,
-        child: const VectorGraphic(
-          loader: AssetBytesLoader('foo.svg'),
-        ),
-      ),
-    );
-    await tester.pumpAndSettle();
-
-    expect(find.byType(RepaintBoundary), findsOneWidget);
-  });
-
```

Comment:
We should add some new tests

Reply:
yeah, let me move the private classes to a src file so we can actually unit test them